.circleci/config.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
```

Comment:
Are you sure this is the correct code for circleCI? Double check this

Reply:
@ZeMunchkin fixed all the major things!